addons/upgrade/to-8.3-conf-latin1-to-utf8.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+find /usr/local/pf/conf -type f -print0 | while read -d $'\0' file; do 
```

Comment:
Also, we need to do this for the templates in html/captive-portal/profile-templates